RELEASES.md

```diff
@@ -22,9 +22,6 @@ for a complete commit history.
 - Add new policy parameter that changes the stacking order of child/dependent credits
   [[#1676](https://github.com/open-source-economics/Tax-Calculator/pull/1676)
   by Matt Jensen as suggested by Cody Kallen]
```

Comment:
Could this read: 
by Matt Jensen as suggested by Cody Kallen and with need identified by JEC staff.

Reply:
or better yet, 'Joint Economic Committee staff'